### PR TITLE
adding new example showing the progress bar feature to change color programmatically

### PR DESCRIPTION
**Pull request by jposada202020**
As described

Expected result when using with a Pyportal

### CODE REVIEW

**Comment by jposada202020** [comment]
@hugodahl if you could review as discussed :)

File: examples/requirements.txt [hunk 1]
```diff
@@ -9,3 +9,4 @@
 Adafruit-Blinka
 adafruit-blinka-displayio
 blinka-displayio-pygamedisplay
+adafruit-fancyled
```
Comment (hugodahl) [inline examples/requirements.txt]: This library failed to install when I tried. It should be referenced as `adafruit-circuitpython-fancyled`

**Review by hugodahl — COMMENTED** [review]
Aside from the 1 change in the examples' `requirements.txt`, looks good to me and works as expected when tested

File: examples/requirements.txt (hunk 1 above)
Comment (jposada202020) [reply examples/requirements.txt]: Thank you, I was doubtful. I check in my packages and your reference did appear available. Solved in https://github.com/adafruit/Adafruit_CircuitPython_ProgressBar/pull/29/commits/a6c291c666a0f8eb34f421a4372e35fa46d58a24

Comment (hugodahl) [reply examples/requirements.txt]: Excellent! Great job :)

**Comment by hugodahl** [comment]
New example works as expected and looks good to me!

**Review by hugodahl — APPROVED** [review]

**Comment by jposada202020** [comment]
Thanks @hugodahl for reviewing. Hopefully we could use this amazing progress bar in more projects :)

**Pull request merged**